lib/src/components/buttons/details_button.dart

```diff
@@ -1,24 +1,40 @@
 import 'package:flutter/material.dart';
 import 'package:help_ukraine_widget/src/components/chevron_down.dart';
-import 'package:help_ukraine_widget/src/theme/font_config.dart';
 
-/// It's a button that changes color when you hover over it
+/// It's a button that changes color when you hover over it.
 class DetailsButton extends StatelessWidget {
   /// It's a function that calls on button tap.
   final VoidCallback onTap;
 
-  /// It's a color that changes appearance of text and icon
+  /// It's a color that changes appearance of text and icon.
   final Color color;
 
-  /// It's a variable that stores the title of the button.
-  final String title;
+  /// how much free space should be between title and chevron.
+  final double spacingToChevron;
 
-  ///Constructor
+  /// how much free space should be after chevron.
```

Comment:
Let's remove DetailsButton completely. I think it makes sense to construct it separately for each widget variation.